tests/utils/test_provider_id_from_name.py

```diff
@@ -0,0 +1,34 @@
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+def test_provider_id_from_name():
+    # positive cases
+    _test_provider("Rite Aid Pharmacy 3897", "rite_aid", "3897")
+    _test_provider("Walgreens #24295", "walgreens", "24295")
+    _test_provider("Walgreens Pharmacy #24295", "walgreens", "24295")
+    _test_provider("Walgreens Specialty Pharmacy #35326", "walgreens", "35326")
+    _test_provider("Walgreens Specialty #24295", "walgreens", "24295")
+    _test_provider("Safeway #85", "safeway", "85")
+    _test_provider("Safeway PHARMACY #85", "safeway", "85")
+    _test_provider("Vons Pharmacy #675", "vons", "675")
+    _test_provider("SAV-ON PHARMACY #585", "albertsons", "585")
+    _test_provider("SAVON PHARMACY #585", "albertsons", "585")
+    _test_provider("Pavilions PHARMACY #8545", "pavilions", "8545")
+    _test_provider("Walmart PHARMACY 10-585", "walmart", "585")
+    _test_provider("Cvs 104", "cvs", "4")  # <== TODO: !!fix!!
```

Comment:
Could you open an issue for this?

Reply:
Will do!

Reply:
Opened: https://github.com/CAVaccineInventory/vaccine-feed-ingest/issues/300